### ISSUE TITLE
Sort lines case insensitive

### ISSUE THREAD

**Issue by mlatushko** [notepad-plus-plus/notepad-plus-plus#4424]
### Description of the Issue
New feature:

### Steps to Reproduce the Issue
-Sort lines case insensitive- at -Line operations- menu

### Expected Behavior
Regular sorting:
A
A
abc
aBd
B
B
b
b

### Actual Behavior
is like:
(Lexicographically / Integer / Decimals Ascending)
A
A
B
B
aBd
abc
b
b

**Comment by sasumner** [comment]
A large variety of sorting options doesn't seem to be a priority for Notepad++ development.  Suggestion:  Do what I did and script something; or create a mini-wrapper around CMD.exe's sort command

**Comment by mlatushko** [comment]
sasumner, so this feature will never be produced?

**Comment by sasumner** [comment]
@mlatushko I don't know the answer to that; I was only offering up options (which you probably already knew).  Your issue remains as a placeholder, and if the developers choose to implement it, then this issue will likely be tagged with that activity.

**Comment by tcloud43** [comment]
For the simple example given, internally converting to upper or lower case in order to perform the sort would solve the problem.  This would probably require converting to an array of two columns, converting all of one column to the same case, sorting, and then providing the other column (unchanged case) as the result.  What am I missing?

**Comment by sasumner** [comment]
@tcloud43 

In a real world scenario, it may not be possible/reasonable to convert text to all one case before doing more (e.g. sorting it).  Sure for a simple example that works.  But I think _that_ is what you are missing.

**Comment by mlatushko** [comment]
Notepad++ should include TextFX plugin in software package. It is so necessary.

**Comment by MetaChuh** [comment]
@mlatushko 

TextFX is available at the notepad++ menu `plugins > plugins admin` but only for 32 bit versions of notepad++, because the original developer never compiled it for 64 bit.

please understand, that we can not implement all possible features directly into the notepad++ core, as different notepad++ users will have different needs.

we usually do not integrate features, that exist and work perfectly using one of the many available plugins. especially because notepad++ relies on the variety of plugin developers.

we are very glad, that the sorting feature of textfx suited your needs for this issue.

thank you for your help and best regards.

-----
```
general notes:

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

for general or plugin questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
```
[<kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd>](https://notepad-plus-plus.org/community/)

**Comment by sasumner** [comment]
Duplicate of #4479 (this one is earlier but that one is still open while this one is currently closed)

**Comment by sasumner** [comment]
The expected behavior from this issue has been implemented in #8869
